src/codegen.rs

```diff
@@ -112,7 +121,11 @@ impl<W: Write> GerberCode<W> for ExtendedCode {
                 writeln!(writer, "*%")?;
             }
             ExtendedCode::DeleteAttribute(ref attr) => {
-                writeln!(writer, "%TD{}*%", attr)?;
+                write!(writer, "%TD")?;
+                if !attr.is_empty() {
+                    write!(writer, ".{}", attr)?;
```

Comment:
the addition of a `.` here causes the `serialization_tests::test_delete_attribute` test to fail.

gerber spec doesn't REQUIRE a leading `.`, some KNOWN attributes start with a `.`, but users can create their own attributes with whatever they like in them.

See 2024.05 5.1 attributes overview:

> In accordance with the general rule in 3.4.3 standard attribute names must begin with a dot ‘.’ while user attribute names cannot begin with a dot. The dot, if present, is part of the attribute name and indicates that it is a standard attribute whose syntax and semantics are defined in section 5.6.